srml/aura/src/lib.rs

```diff
@@ -60,6 +60,8 @@ use timestamp::TimestampInherentData;
 use inherents::{RuntimeString, InherentIdentifier, InherentData, ProvideInherent, MakeFatalError};
 #[cfg(feature = "std")]
 use inherents::{InherentDataProviders, ProvideInherentData};
+#[cfg(feature = "std")]
```

Comment:
I don't think that these 2 lines are needed

Reply:
Indeed!